src/System.Threading.Tasks.Dataflow/src/System.Threading.Tasks.Dataflow.csproj

```diff
@@ -6,6 +6,7 @@
     <RootNamespace>System.Threading.Tasks.Dataflow</RootNamespace>
     <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
     <DefineConstants Condition="'$(TargetGroup)' == 'netstandard1.1' OR '$(TargetGroup)' == 'netstandard'">$(DefineConstants);CONCURRENT_COLLECTIONS;FEATURE_TRACING</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netstandard'">$(DefineConstants);THREADING</DefineConstants>
```

Comment:
If we reverse the sense of the define, and define it for 1.0/1.1 only, perhaps it is clearer that it is a poly-fill just for those old targets?

Actually perhaps that is what is suggested by the name USE_INTERNAL_THREADING